web3j/pom.xml

```diff
@@ -36,6 +36,11 @@
       <artifactId>spring-boot-starter-validation</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
Considering the simplicity of the requirement, JPA is an overkill. Checkout https://www.sql2o.org/

Reply:
I was checking out sql2o, actually, ramki has used hstore data type, in JPA it was simple to implement it in contract model. But for sql2o I will need to figure it out.